SEED-platform/seed
# Feat/derived column inventory list

#### Any background context you want to provide?
SEED is adding the ability for users to define columns which are evaluated based on other columns.

#### What's this PR do?
1. shows derived column values in the inventory list and inventory detail page
2. allows derived columns to be added to column list and detail profiles

#### How should this be manually tested?
1. create some derived columns from the organization section.
2. create/update column profile to include the derived columns
3. update/confirm one or more properties has all data required for these derived columns (so the value will not be null after evaluating)
4. view the inventory list and inventory detail page to see the results. You should not be able to edit derived column values.

#### What are the relevant tickets?
#2710 

#### Screenshots (if appropriate)
![Screen Shot 2021-06-11 at 5 25 19 PM](https://user-images.githubusercontent.com/18518728/121757548-3291da00-cadb-11eb-8531-2ae7b9895360.png)
![Screen Shot 2021-06-11 at 5 24 55 PM](https://user-images.githubusercontent.com/18518728/121757549-332a7080-cadb-11eb-8bbb-edfe18cb8d94.png)
![Screen Shot 2021-06-11 at 5 24 20 PM](https://user-images.githubusercontent.com/18518728/121757551-332a7080-cadb-11eb-99b2-3c00e1fc66a4.png)
![Screen Shot 2021-06-11 at 5 24 01 PM](https://user-images.githubusercontent.com/18518728/121757552-332a7080-cadb-11eb-88a9-cb70252bf109.png)
